angular/src/components/settings/vault-timeout-input.component.ts

```diff
@@ -19,6 +19,8 @@ export class VaultTimeoutInputComponent implements ControlValueAccessor, Validat
   }
 
   static CUSTOM_VALUE = -100;
+  static MIN_CUSTOM_MINUTES = 1;
+  validMinutes = true;
```

Comment:
This is a custom input component which implements `ControlValueAccessor`. For validation purposes you can use the `validate` function below.

Reply:
okay I would effect the change